index.html

```diff
@@ -565,9 +565,9 @@ <h1 class="text-white">Wordpress Plugins</h1>
 									<li>
 										<h1>Susper Search Engine</h1>
 										<p class="lead">
-											Susper is a decentralised search engine using the peer to peer search technology of yacy and search engine applications like Apache Solr. <br>[Technologies used: Yacy, Apache Solr, Backbone.js Javascript, HTML, CSS, JSON]<br>
+											Susper is a decentralised search engine using the peer to peer search technology of yacy and search engine applications like Apache Solr. <br>[Technologies used: Yacy, Apache Solr, Angular2 ,TypeScript, HTML, CSS, JSON]<br>
```

Comment:
A simple indentation a space after comma seems silly but this one of the most read repos :)